.storybook/themes/news.scss

```
@@ -685,6 +685,41 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
+					background: var(--background-color),
+					box-shadow: var(--global-box-shadow-1),
+					border-radius: 0 var(--border-radius) var(--border-radius) 0,
+					display: none,
+					left: 0,
+					padding: var(--global-spacing-4) 0,
+					position: fixed,
+					top: 200px,
+					width: 44px,
+					components: (
+						button-medium: (
+							background: transparent,
+							border-width: 0,
+							padding: 0 10px,
+							text-align: center,
+							width: 44px,
+						),
+					),
+				),
+				share-bar-facebook: (
+					fill: #415993,
+				),
+				share-bar-linkedin: (
+					fill: #3076b0,
+				),
+				share-bar-pinterest: (
+					fill: #ad2526,
+				),
+				share-bar-twitter: (
+					fill: #6baae8,
+				),
+				share-bar-envelope: (
```

As per suggestion in the CSS

```suggestion
				share-bar-email: (
```